src/css.js

```diff
@@ -130,11 +130,12 @@ function boxModelAdjustment( elem, dimension, box, isBorderBox, styles, computed
 
 	// Account for positive content-box scroll gutter when requested by providing computedVal
 	if ( !isBorderBox && computedVal >= 0 ) {
+		var val = elem[ "offset" + dimension[ 0 ].toUpperCase() + dimension.slice( 1 ) ];
```

Comment:
I agree with @mgol on comment placement, and also observe that
* defining a new variable here is against our general style, and
* `val` is an overly generic name, and
* it provides little benefit anyway, since `delta` will never be modified when offset* is unknown (`computedVal - computedVal - delta - extra - 0.5` is guaranteed to be negative in this case, so `Math.ceil` will not be positive, so `Math.max` will be 0).

I think we should keep things simpler:
```js
// offsetWidth/offsetHeight is a rounded sum of content, padding, scroll gutter, and border
// Scroll gutter is that value minus the other properties, rounded half-down to the nearest integer
delta += Math.max( 0, Math.ceil(
	elem[ "offset" + dimension[ 0 ].toUpperCase() + dimension.slice( 1 ) ] -
	computedVal -
	delta -
	extra -
	0.5

// If offsetWidth/offsetHeight is unknown, then we can't determine content-box scroll gutter
// Use an explicit zero to avoid NaN (gh-3964)
) ) || 0;
```